Docs/reference/requests_and_deploys.md

```diff
@@ -0,0 +1,73 @@
+# Requests and Deploys
+
+Singularity uses the concepts of a `Request` and `Deploy` to keep track of changes to a particular task or set of related tasks.
+
+A `Request` can be thought of as the high level information for a single project or deployable item. For example, the name and owner's email for a single web service, or the name and cron schedule for a single scheduled job.
+
+A `Deploy` can then be thought of as the specific configuration or version of the running code for that deployable item.
+
+To illustrate the differences between and usages of these two concepts, we will walk through an example using Singularity to run a single web service.
+
+## The `TestService` Example
+
+
+### Creating a `Request`
+You have a new web service called `TestService` that you want to run via Singularity. The first thing you need to do is create a `Request` for `TestService`. To create this request, you would `POST` json over http to the Singularity API or create a request via the new request page in the Singularity UI. Example json:
```

Comment:
may want to include the endpoint (`.../api/requests`) or a link to the API reference doc for creating the request